src/more-infos/more-info-light.html

```diff
@@ -40,9 +40,19 @@
           on-change='brightnessSliderChanged' class='flex'>
         </paper-slider>
       </div>
+      <template is="dom-if" if="{{ctSliderValue}}">
+        <div class='brightness center horizontal layout'>
```

Comment:
Change the class name to `color_temperature` and in the CSS above, add a similar construction as brightness. You can then remove the `template` IF statement and the slider will fade in when a new state arrives that has a temperature.

You can change line 8 to be `.brightness, .color_temperature {`